### ISSUE TITLE
JS broken in upgrade/fresh install

### ISSUE THREAD

**Issue by scottbert** [Strider-CD/strider#855]
```
Uncaught SyntaxError: Unexpected token *
plugin-config-compiled.js:1 Uncaught ReferenceError: angular is not defined(anonymous function) @ plugin-config-compiled.js:1
account-plugins-compiled.js:1 Uncaught ReferenceError: angular is not defined(anonymous function) @ account-plugins-compiled.js:1
plugin-status-compiled.js:1 Uncaught ReferenceError: angular is not defined
```

...and I can't see anything other than the menu when I visit the page. Originally this was an upgrade, I've tried everything I can think of including updating to  node .12 from node .10, removing the entire of my node_modules and reinstalling as well as also "npm run build" in Strider's node_modules directory.

The error is at line 431. After `npm run build` the error shifts one line to 430. Same issue.

The code is here: 
```
function $TemplateCacheProvider(){this.$get=['$cacheFactory',function($cacheFactory){return $cacheFactory('templates');}];}
dResourceUrl}.
 *
 *
 * #### `replace` ([*DEPRECATED*!], will be removed in next major release)
```
Looks like a comment is being incorrectly processed.
Full file is here: https://ci.freakcity.net/scripts/app.js

Any ideas?

**Comment by knownasilya** [comment]
So your code looks fine at that example that you pasted when looking at your script. Also your site is showing more then just the menu. Is this solved?

**Comment by scottbert** [comment]
No, sorry. I reverted to 1.6.6 as I needed to do some builds. So I have a working 1.6.6, do you have upgrade instructions for 1.7, I can try again over the weekend?

**Comment by knownasilya** [comment]
Upgrade to 1.7.4 should just be an npm install, since the build is already in the repo.

**Comment by scottbert** [comment]
OK. Just updated to the latest. Similar issue - you can see if you check out the url.

Fedora 22 x86_64, Node 0.12.7, NPM 2.11.3.

Full upgrade log below
```
npm WARN deprecated swig@0.14.0: v1.0.0 is a complete rewrite of Swig from the ground up. Previous versions are no longer supported
npm WARN deprecated MD5@1.3.0: deprecated, use lowercase 'md5@2.x' from now on
npm WARN engine global-tunnel@1.2.0: wanted: {"node":"0.10.x"} (current: {"node":"0.12.7","npm":"2.11.3"})
npm WARN optional dep failed, continuing fsevents@0.3.0
npm WARN engine hawk@0.10.2: wanted: {"node":"0.8.x"} (current: {"node":"0.12.7","npm":"2.11.3"})
npm WARN engine cryptiles@0.1.3: wanted: {"node":"0.8.x"} (current: {"node":"0.12.7","npm":"2.11.3"})
npm WARN engine boom@0.3.8: wanted: {"node":"0.8.x"} (current: {"node":"0.12.7","npm":"2.11.3"})
npm WARN engine sntp@0.1.4: wanted: {"node":"0.8.x"} (current: {"node":"0.12.7","npm":"2.11.3"})
npm WARN engine hoek@0.7.6: wanted: {"node":"0.8.x"} (current: {"node":"0.12.7","npm":"2.11.3"})
 
> utf-8-validate@1.1.0 install /usr/lib/node_modules/strider/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/utf-8-validate
> node-gyp rebuild

make: Entering directory '/usr/lib/node_modules/strider/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/utf-8-validate/build'
  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/obj.target/validation.node
  COPY Release/validation.node
make: Leaving directory '/usr/lib/node_modules/strider/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/utf-8-validate/build'
 
> bufferutil@1.1.0 install /usr/lib/node_modules/strider/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/bufferutil
> node-gyp rebuild

make: Entering directory '/usr/lib/node_modules/strider/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/bufferutil/build'
  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/obj.target/bufferutil.node
  COPY Release/bufferutil.node
make: Leaving directory '/usr/lib/node_modules/strider/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/bufferutil/build'
-
> utf-8-validate@1.1.0 install /usr/lib/node_modules/strider/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_modules/utf-8-validate
> node-gyp rebuild

make: Entering directory '/usr/lib/node_modules/strider/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_modules/utf-8-validate/build'
  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/obj.target/validation.node
  COPY Release/validation.node
make: Leaving directory '/usr/lib/node_modules/strider/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_modules/utf-8-validate/build'
 
> bufferutil@1.1.0 install /usr/lib/node_modules/strider/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_modules/bufferutil
> node-gyp rebuild

make: Entering directory '/usr/lib/node_modules/strider/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_modules/bufferutil/build'
  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/obj.target/bufferutil.node
  COPY Release/bufferutil.node
make: Leaving directory '/usr/lib/node_modules/strider/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_modules/bufferutil/build'

> kerberos@0.0.12 install /usr/lib/node_modules/strider/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos
> (node-gyp rebuild 2> builderror.log) || (exit 0)

make: Entering directory '/usr/lib/node_modules/strider/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos/build'
  CXX(target) Release/obj.target/kerberos/lib/kerberos.o
  CXX(target) Release/obj.target/kerberos/lib/worker.o
  CC(target) Release/obj.target/kerberos/lib/kerberosgss.o
  CC(target) Release/obj.target/kerberos/lib/base64.o
  CXX(target) Release/obj.target/kerberos/lib/kerberos_context.o
  SOLINK_MODULE(target) Release/obj.target/kerberos.node
  COPY Release/kerberos.node
make: Leaving directory '/usr/lib/node_modules/strider/node_modules/connect-mongo/node_modules/mongodb/node_modules/mongodb-core/node_modules/kerberos/build'

> kerberos@0.0.11 install /usr/lib/node_modules/strider/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos
> (node-gyp rebuild 2> builderror.log) || (exit 0)

make: Entering directory '/usr/lib/node_modules/strider/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos/build'
  CXX(target) Release/obj.target/kerberos/lib/kerberos.o
  CXX(target) Release/obj.target/kerberos/lib/worker.o
  CC(target) Release/obj.target/kerberos/lib/kerberosgss.o
  CC(target) Release/obj.target/kerberos/lib/base64.o
  CXX(target) Release/obj.target/kerberos/lib/kerberos_context.o
  SOLINK_MODULE(target) Release/obj.target/kerberos.node
  COPY Release/kerberos.node
make: Leaving directory '/usr/lib/node_modules/strider/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos/build'

> bson@0.2.22 install /usr/lib/node_modules/strider/node_modules/mongoose/node_modules/mongodb/node_modules/bson
> (node-gyp rebuild 2> builderror.log) || (exit 0)

make: Entering directory '/usr/lib/node_modules/strider/node_modules/mongoose/node_modules/mongodb/node_modules/bson/build'
  CXX(target) Release/obj.target/bson/ext/bson.o
  SOLINK_MODULE(target) Release/obj.target/bson.node
  COPY Release/bson.node
make: Leaving directory '/usr/lib/node_modules/strider/node_modules/mongoose/node_modules/mongodb/node_modules/bson/build'
/usr/bin/strider -> /usr/lib/node_modules/strider/bin/strider
strider@1.7.4 /usr/lib/node_modules/strider
├── cookie@0.1.3
├── connect-flash@0.1.1
├── connect-slashes@1.3.1
├── gravatar@1.3.0
├── eventemitter2@0.4.14
├── extend@2.0.1
├── step@0.0.5
├── thirty-two@0.0.2
├── everypaas@0.0.7
├── async@1.4.2
├── cookie-parser@1.3.5 (cookie-signature@1.0.6)
├── serve-favicon@2.3.0 (fresh@0.3.0, ms@0.7.1, etag@1.7.0, parseurl@1.3.0)
├── path-to-regexp@1.2.1 (isarray@0.0.1)
├── method-override@2.3.5 (methods@1.1.1, vary@1.0.1, parseurl@1.3.0)
├── passport-local@1.0.0 (passport-strategy@1.0.0)
├── strider-build-badge@0.1.0
├── MD5@1.3.0 (charenc@0.0.1, crypt@0.0.1)
├── debug@2.2.0 (ms@0.7.1)
├── ssh-keypair@2.0.0 (crypto@0.0.3)
├── validator@3.43.0
├── passport@0.2.2 (pause@0.0.1, passport-strategy@1.0.0)
├── cors@2.7.1 (vary@1.0.1)
├── morgan@1.6.1 (basic-auth@1.0.3, on-headers@1.0.0, depd@1.0.1, on-finished@2.3.0)
├── rc@1.1.1 (ini@1.3.4, strip-json-comments@0.1.3, deep-extend@0.2.11, minimist@1.2.0)
├── semver@4.3.6
├── express-session@1.11.3 (utils-merge@1.0.0, cookie-signature@1.0.6, on-headers@1.0.0, parseurl@1.3.0, depd@1.0.1, crc@3.3.0, uid-safe@2.0.0)
├── ansi_up@1.3.0
├── cli-table@0.3.1 (colors@1.0.3)
├── bcryptjs@2.2.1
├── connect@3.4.0 (utils-merge@1.0.0, parseurl@1.3.0, finalhandler@0.4.0)
├── mailer@0.6.7 (colors@1.1.2, nodemailer@0.1.20)
├── strider-git@0.2.4 (shell-escape@0.0.1, gitane@0.3.2, fs-extra@0.8.1)
├── swig@0.14.0 (underscore@1.8.3)
├── strider-custom@0.6.0 (gitane@0.3.2, ejs@1.0.0)
├── errorhandler@1.4.2 (escape-html@1.0.2, accepts@1.2.13)
├── compression@1.5.2 (bytes@2.1.0, vary@1.0.1, on-headers@1.0.0, compressible@2.0.5, accepts@1.2.13)
├── superagent@1.3.0 (extend@1.2.1, methods@1.0.1, cookiejar@2.0.1, component-emitter@1.1.2, reduce-component@1.0.1, mime@1.3.4, qs@2.3.3, readable-stream@1.0.27-1, formidable@1.0.14, form-data@0.2.0)
├── body-parser@1.13.3 (bytes@2.1.0, content-type@1.0.1, depd@1.0.1, on-finished@2.3.0, http-errors@1.3.1, raw-body@2.1.2, qs@4.0.0, type-is@1.6.8, iconv-lite@0.4.11)
├── bluebird@2.10.0
├── express@4.13.3 (escape-html@1.0.2, merge-descriptors@1.0.0, array-flatten@1.1.1, utils-merge@1.0.0, cookie-signature@1.0.6, methods@1.1.1, fresh@0.3.0, range-parser@1.0.2, vary@1.0.1, path-to-regexp@0.1.7, content-type@1.0.1, etag@1.7.0, parseurl@1.3.0, content-disposition@0.5.0, serve-static@1.10.0, depd@1.0.1, finalhandler@0.4.0, on-finished@2.3.0, qs@4.0.0, proxy-addr@1.0.8, send@0.13.0, type-is@1.6.8, accepts@1.2.13)
├── strider-heroku@0.1.1 (ssh-keypair@1.0.1, strider-git@0.1.1, passport-heroku@0.0.2, superagent@0.15.7)
├── strider-ruby@0.0.2 (strider-detection-rules@0.0.1)
├── winston@1.0.1 (cycle@1.0.3, stack-trace@0.0.9, eyes@0.1.8, isstream@0.1.2, pkginfo@0.3.0, async@0.9.2, colors@1.0.3)
├── strider-python@0.2.1 (strider-detection-rules@0.0.1)
├── nodemailer@0.6.5 (public-address@0.1.1, directmail@0.1.8, he@0.3.6, readable-stream@1.1.13, simplesmtp@0.3.35, mailcomposer@0.2.12)
├── httpcheck@0.2.0 (request@2.16.6)
├── request@2.61.0 (aws-sign2@0.5.0, forever-agent@0.6.1, caseless@0.11.0, stringstream@0.0.4, oauth-sign@0.8.0, tunnel-agent@0.4.1, form-data@1.0.0-rc3, isstream@0.1.2, json-stringify-safe@5.0.1, extend@3.0.0, combined-stream@1.0.5, node-uuid@1.4.3, qs@4.0.0, mime-types@2.1.6, http-signature@0.11.0, bl@1.0.0, har-validator@1.8.0, tough-cookie@2.0.0, hawk@3.1.0)
├── strider-mailer@0.2.1 (lodash.foreach@2.4.1, nodemailer@0.5.15)
├── strider-slack@2.0.0 (ejs@1.0.0, lodash.foreach@2.4.1, slackihook@1.0.0, lodash@2.4.2)
├── strider-bitbucket@1.2.0 (debug@0.7.4, gravatar@1.0.6, crypto@0.0.3, async@0.2.10, passport-bitbucket@0.1.2, oauth@0.9.13, passport@0.1.18, superagent@0.15.7, lodash@2.2.1, request@2.27.0)
├── strider-ssh-deploy@0.4.2 (ejs@1.0.0, progress-stream@0.5.0, ssh2@0.3.6, lodash@2.4.2, npmd-pack@1.0.0)
├── strider-env@0.5.1 (lodash@1.3.1)
├── strider-metadata@0.0.3 (lodash@1.3.1)
├── strider-github@2.1.1 (ssh-keypair@1.0.1, debug@0.7.4, scmp@0.0.2, gravatar@1.0.6, crypto@0.0.3, async@0.2.10, passport@0.1.18, passport-github@1.0.0, superagent@0.15.7, lodash@2.4.2)
├── global-tunnel@1.2.0 (tunnel@0.0.2, lodash@1.3.1)
├── strider-simple-runner@0.13.1 (async@0.1.22, dirkeeper@0.2.0, fs-extra@0.8.1, strider-runner-core@2.0.0, lodash@1.3.1)
├── socket.io@1.3.6 (debug@2.1.0, has-binary-data@0.1.3, socket.io-adapter@0.3.1, engine.io@1.5.2, socket.io-parser@2.2.4)
├── socket.io-client@1.3.6 (to-array@0.1.3, indexof@0.0.1, component-bind@1.0.0, debug@0.7.4, backo2@1.0.2, object-component@0.0.3, component-emitter@1.1.2, has-binary@0.1.6, parseuri@0.0.2, engine.io-client@1.5.2, socket.io-parser@2.2.4)
├── lodash@3.10.1
├── jade@0.35.0 (character-parser@1.2.0, commander@2.0.0, mkdirp@0.3.5, with@1.1.1, transformers@2.1.0, constantinople@1.0.2, monocle@1.1.50)
├── strider-email-notifier@0.4.2 (underscore@1.5.2, async@0.2.10, moment@2.4.0)
├── strider-cli@1.4.10 (pw@0.0.4, spawn-cmd@0.0.2, chokidar@0.9.0, touch@0.0.3, nomnom@1.8.1, glob@4.5.3, rimraf@2.4.3, superagent@0.20.0, prompt@0.2.14, lodash@2.4.2, strider-ecosystem-client@1.2.1)
├── moment@2.10.6
├── connect-mongo@0.8.2 (depd@1.0.1, mongodb@2.0.42)
├── strider-extension-loader@0.4.5 (methods@0.0.1, underscore@1.5.2, async@0.2.10, connect@2.4.6, less@1.4.2)
├── mongoose@3.8.35 (regexp-clone@0.0.1, sliced@0.0.5, muri@1.1.0, hooks@0.2.1, mpath@0.1.1, mpromise@0.4.3, ms@0.1.0, mquery@1.6.1, mongodb@1.4.38)
└── strider-node@0.6.5 (n@1.4.2, async@0.2.10, fs-extra@0.8.1, npm@2.14.3)
```

**Comment by knownasilya** [comment]
Looks like you have some weird issue, because now it's broken at a different place in the script. Plus your script doesn't line up what's here: https://github.com/Strider-CD/strider/tree/v1.7.4

**Comment by scottbert** [comment]
Yes, I have some weird issue, is why I raised a bug :P 
Script doesn't always break in exactly the same place - previously I had Strider 1.7.3, now is 1.7.4. 
If I go into the strider directory and run `nam run build` it'll break somewhere similar but not quite the same. I'm guessing because the versions of the scripts being compiled have changed.

The error is always consistent though. It breaks because it looks like the top of a comment has been chopped off - as if minification has just stopped working at some point.

Package JSON is:
```
{
  "name": "strider",
  "version": "1.7.4",
  "bin": {
    "strider": "bin/strider"
  },
  "main": "main.js",
  "description": "Brilliant continuous deployment platform",
  "license": "BSD-3-Clause",
  "keywords": [
    "strider",
    "continuous integration",
    "continuous deployment",
    "testing",
    "deployment"
  ],
  "engines": {
    "node": "> 0.8.x",
    "npm": ">1.4.0 <3.0.0"
  },
  "repository": {
    "type": "git",
    "url": "git://github.com/Strider-CD/strider.git"
  },
  "browser": {
    "bootstrap": "./vendor/bootstrap/js/bootstrap.js",
    "sortable": "./vendor/Sortable/js/Sortable.js",
    "bootbox": "./vendor/bootbox/js/bootbox.js",
    "codemirror": "./vendor/CodeMirror/js/codemirror.js",
    "codemirror-shell": "./vendor/CodeMirror/js/shell.js",
    "timeago": "./vendor/jquery-timeago/js/jquery.timeago.js",
    "ui-codemirror": "./public/libs/ui-codemirror.js",
    "ui-bootstrap": "./public/libs/ui-bootstrap.js",
    "ui-sortable": "./public/libs/ui-sortable.js"
  },
  "browserify-shim": "./shim.js",
  "browserify": {
    "transform": [
      "browserify-shim"
    ]
  },
  "scripts": {
    "test": "make test",
    "start": "bin/strider",
    "dev": "npm run build-debug && bin/strider",
    "pre": "mkdirp dist/scripts dist/styles",
    "vendor": "bower-installer",
    "styles": "lessc client/styles/strider.less > dist/styles/styles.css",
    "styles-debug": "lessc --source-map client/styles/strider.less dist/styles/styles.css",
    "watch": "npm run pre && watchify -e client/app.js -o dist/scripts/app.js",
    "build": "npm run pre && npm run styles && browserify -t [babelify --only client] -e client/app.js -o dist/scripts/app.js",
    "build-debug": "npm run pre && npm run styles-debug && browserify -d -e client/app.js -o dist/scripts/app.js",
    "unitcov": "istanbul cover _mocha --recursive test/unit",
    "unitdev": "mocha -w --recursive test/unit -R min",
    "gendocs": "apidoc -i lib/ -o apidocs/"
  },
  "dependencies": {
    "MD5": "^1.2.1",
    "ansi_up": "^1.3.0",
    "async": "^1.2.1",
    "bcryptjs": "^2.1.0",
    "bluebird": "^2.3.2",
    "body-parser": "^1.10.0",
    "cli-table": "^0.3.0",
    "compression": "^1.2.2",
    "connect": "^3.3.5",
    "connect-flash": "^0.1.1",
    "connect-mongo": "^0.8.1",
    "connect-slashes": "^1.3.1",
    "cookie": "^0.1.3",
    "cookie-parser": "^1.3.3",
    "cors": "^2.5.2",
    "debug": "^2.2.0",
    "errorhandler": "^1.3.0",
    "eventemitter2": "0.4.14",
    "everypaas": "0.0.x",
    "express": "^4.10.6",
    "express-session": "^1.9.3",
    "extend": "^2.0.0",
    "global-tunnel": "^1.2.0",
    "gravatar": "^1.1.0",
    "httpcheck": "^0.2.0",
    "jade": "^0.35.0",
    "lodash": "^3.9.3",
    "mailer": "^0.6.7",
    "method-override": "^2.3.0",
    "moment": "^2.8.3",
    "mongoose": "3.8.*",
    "morgan": "^1.5.0",
    "nodemailer": "^0.6.5",
    "passport": "^0.2.0",
    "passport-local": "^1.0.0",
    "path-to-regexp": "^1.0.2",
    "rc": "^1.0.3",
    "request": "^2.49.0",
    "semver": "^4.3.3",
    "serve-favicon": "^2.2.0",
    "socket.io": "^1.2.1",
    "socket.io-client": "^1.0.6",
    "ssh-keypair": "^2.0.0",
    "step": "0.0.5",
    "strider-bitbucket": "^1.2.0",
    "strider-build-badge": "^0",
    "strider-cli": "^1.4.1",
    "strider-custom": "^0.6.0",
    "strider-email-notifier": "^0.4.2",
    "strider-env": "^0.5.1",
    "strider-extension-loader": "^0.4.5",
    "strider-git": "^0.2.0",
    "strider-github": "^2.1.1",
    "strider-heroku": "^0.1.1",
    "strider-mailer": "^0.2.0",
    "strider-metadata": "^0.0.3",
    "strider-node": "^0.6.5",
    "strider-python": "^0.2.1",
    "strider-ruby": "^0.0.2",
    "strider-simple-runner": "^0.13.1",
    "strider-slack": "^2.0.0",
    "strider-ssh-deploy": "^0.4.0",
    "superagent": "^1.2.0",
    "swig": "^0.14.0",
    "thirty-two": "0.0.2",
    "validator": "^3.22.1",
    "winston": "^1.0.0"
  },
  "devDependencies": {
    "angular": "1.2.23",
    "angular-route": "1.2.17-build.163.1",
    "apidoc": "^0.13.1",
    "babelify": "^6.1.2",
    "blanket": "^1.1.5",
    "bower-installer": "^1.2.0",
    "browserify": "^6.3.3",
    "browserify-shim": "^3.8.0",
    "chai": "^1.9.2",
    "chai-as-promised": "^4.1.0",
    "commander": ">=0.5.2",
    "expect.js": "^0.3.1",
    "jquery": "^2.1.4",
    "jshint": "^2.5.6",
    "less": "^2.2.0",
    "mkdirp": "^0.5.0",
    "mocha": "^1.21.4",
    "node-mocks-http": "^1.2.4",
    "should": "^2.1.0",
    "sinon": "^1.10.3",
    "watchify": "^0.10.2",
    "wd": "^0.3.8"
  },
  "gitHead": "8016ab720ffee182c5ce38212560514c4ccecffd",
  "bugs": {
    "url": "https://github.com/Strider-CD/strider/issues"
  },
  "homepage": "https://github.com/Strider-CD/strider#readme",
  "_id": "strider@1.7.4",
  "_shasum": "f8c6325631b546e93536a4ae0324c7ef71afac30",
  "_from": "strider@1.7.4",
  "_npmVersion": "2.11.1",
  "_nodeVersion": "2.3.0",
  "_npmUser": {
    "name": "knownasilya",
    "email": "ilya@burstcreations.com"
  },
  "maintainers": [
    {
      "name": "niallo",
      "email": "niallo@beyondfog.com"
    },
    {
      "name": "peterbraden",
      "email": "peterbraden@peterbraden.co.uk"
    },
    {
      "name": "knownasilya",
      "email": "ilya@burstcreations.com"
    }
  ],
  "dist": {
    "shasum": "f8c6325631b546e93536a4ae0324c7ef71afac30",
    "tarball": "http://registry.npmjs.org/strider/-/strider-1.7.4.tgz"
  },
  "directories": {},
  "_resolved": "https://registry.npmjs.org/strider/-/strider-1.7.4.tgz",
  "readme": "ERROR: No README data found!"
}
```

**Comment by knownasilya** [comment]
Maybe give master a try?

**Comment by knownasilya** [comment]
Any luck?

**Comment by knownasilya** [comment]
Were you able to find a reason?

**Comment by knownasilya** [comment]
Might be fixed by be36b4cce8e83f80a69d59e4ebd3e94fa2764a41

**Comment by knownasilya** [comment]
Closing for now, until someone says otherwise.
